languages/lua/lua_test.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"testing"
+)
+
+// Mostly a debugging aid, not sure if it'll actually fail if something goes wrong
+func TestLuaStuff(t *testing.T) {
+	var lua Lua
+	lua.Open()
+	lua.Version()
+	lua.EvalFile("testdata/test.lua", make([]string, 0))
```

Comment:
since this is a race (presumably??), can this be run on a small 100-ish loop? mention the race in a comment somewhere.

bonus points if reverting the fix causes this to fail!

Reply:
I don't think it's a race so much as just a memory layout issue. With the test there's low enough overall entropy that it fails *every time* without the patch, but when testing through the web UI the rest of the container's contents is running enough to shift it around and only cause intermittent crashes.

It works alright in a basic loop, but becomes _very unhappy_ when that loop is spawning concurrent goroutines, though I think this is more the result of Lua's c api not being thread safe. (Do we have anything up the stack that ensures everything is called on one thread?)

Reply:
Alright, looks like it does lock the thread [here](https://github.com/replit/prybar/blob/master/utils/utils.go#L47) so concurrency isn't an issue.

@lhchavez any last thoughts on this? tl;dr I think it's memory layout, not a race condition.

Reply:
:shipit: